lti/views/module_item_configurations.py

```diff
@@ -1,17 +1,17 @@
 from pyramid.view import view_config
 from pyramid.httpexceptions import HTTPFound
 from lti.util.lti_launch import lti_launch
-import pdb
+from lti.models.module_item_configuration import ModuleItemConfiguration
 
-@view_config(route_name='module_item_configurations', renderer='lti:templates/module_item_configurations/new_module_item_configuration.html.jinja2', request_method='POST')
-@lti_launch
-def new_module_item_configuration(request):
-  """
-    Renders the form that teachers see to configure the module item.
-  """
+@view_config(route_name='module_item_configurations', renderer='lti:templates/lti_launches/create_lti_launch.html.jinja2', request_method='POST')
+def create_module_item_configuration(request):
+  # TODO Verify the user is actually logged in!
+  instance = ModuleItemConfiguration(
+    document_url=request.params['document_url'],
```

Comment:
Do we need to deal with possibility of these keys not being present?

Reply:
the document url probably. We will want to handle the form validation on the front end which would be best implemented when we get stuff from brandon. I will add a TODO for that.